R/nse-defuse.R

```diff
@@ -117,6 +117,8 @@ expr <- function(expr) {
 #'   of `"trailing"`, `"none"`, `"all"`. If `"trailing"`, only the
 #'   last argument is ignored if it is empty. Named arguments are not
 #'   considered empty.
+#' @param .ignore_null Whether to ignore null arguments. Can be
```

Comment:
Would this be clearer if it was "unnamed" or "none" ?

Reply:
I think so. However it would incorrectly imply that `.ignore_empty = "all"` behaves differently than `.ignore_null = "unnamed"`.